changes/ce/feat-12483.en.md

```diff
@@ -0,0 +1,2 @@
+Renamed `emqx ctl conf cluster_sync tnxid ID` to `emqx ctl conf cluster_sync inspect ID`.
```

Comment:
Nit: `breaking-12483.en.md` ?

It's not breaking yet, since the fallback is present, but it will become in the future.